tests/validation/tests/v3_api/common.py

```diff
@@ -1503,3 +1503,35 @@ def rbac_cleanup():
     client.delete(rbac_data["wl_unshared"])
     client.delete(rbac_data["p_unshared"])
 
+
+def wait_for_wl_by_id_to_active(client, wl, timeout=DEFAULT_TIMEOUT):
+    start = time.time()
+    workload = client.list_workload(id='deployment:'
+                                    + wl['metadata']['namespace']
+                                    + ':' + wl['metadata']['name']).data[0]
+    
+    while workload.state != "active":
+        if time.time() - start > timeout:
+            raise AssertionError(
+                "Timed out waiting for state to get to active")
+        time.sleep(.5)
+        workload = client.list_workload(id='deployment:'
+                                        + wl['metadata']['namespace'] + ':'
+                                        + wl['metadata']['name']).data[0]
+    return workload
```

Comment:
This function looks like a duplication of `wait_for_wl_to_active(client, workload, timeout=DEFAULT_TIMEOUT)` under `common.py`

Could we reuse the existing code?